### PR TITLE
fix(TBD-11130): ClassNotFoundException when I check service for CDH 6.3.3& CDH 6.1.1 & HDP 3.1 CDH part

### PR DESCRIPTION
**Pull request by Redwene**
### CDH part

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 1]
```diff
@@ -6212,7 +6045,7 @@
         "tagName" : "library"
       }, {
         "childNodes" : [ ],
-        "id" : "DYNAMIC_Cloudera_CDH6_1_1_jettison_1_3_8_jar",
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_jettison_1_1_jar",
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:6048]: strange that we get a lower version than before here

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 2]
```diff
@@ -279,10 +283,6 @@
         "childNodes" : [ ],
         "id" : "DYNAMIC_Cloudera_CDH6_1_1_jackson_databind_2_9_5_jar",
         "tagName" : "library"
-      }, {
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:282]: I don't get why we have all those removals

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 3]
```diff
@@ -3913,6 +3805,11 @@
       "id" : "DYNAMIC_Cloudera_CDH6_1_1_IMPALA_LIB_DYNAMIC",
       "tagName" : "libraryNeededGroup",
       "templateId" : "IMPALA-LIB-DYNAMIC"
+    }, {
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:3808]: I haven't seen this module group added in template. Is it coming from elsewhere ?

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 1 above)
Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:6048]: It's what dynamic distribution generated

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 3 above)
Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:3808]: coming from 
image

Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:3808]: image

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 2 above)
Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:282]: It's what dynamic distribution generated

**Review by CarlosTalend — APPROVED** [review]

**Review by lbourgeois — APPROVED** [review]
LGTM, need to be smoke tested

**Pull request merged**